modules/oadp-creating-restore-cr.adoc

```diff
@@ -55,3 +55,15 @@ $ oc get restore -n openshift-adp <restore> -o jsonpath='{.status.phase}'
 $ oc get all -n <namespace> <1>
 ----
 <1> Namespace that you backed up.
+
+. If you use Restic to restore DeploymentConfigs or if you use post-restore hooks, run a cleanup script by entering the following command:
```

Comment:
s/DeploymentConfigs/`DeploymentConfigs` 

Add backticks to DeploymentConfigs for each reference.

Reply:
@bmcelvee 
Done. But I've changed 'DeploymentConfigs' to '`DeploymentConfig` objects'.